linkerd/app/outbound/src/ingress.rs

```diff
@@ -319,7 +319,9 @@ impl TryFrom<Discovery<Http<RequestTarget>>> for Http<Logical> {
                     }
 
                     // If the client policy is default, try the service profile first
```

Comment:
comment should maybe be updated, since the default-ness of the policy is not directly the condition here.